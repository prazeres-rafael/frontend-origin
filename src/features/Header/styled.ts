import styled from 'styled-components';

export const HeaderOrigin = styled.header({
  backgroundColor: '#FFF',
  height: 56,
  left: 0,
  position: 'relative',
  top: 0,
  width: '100%',

  '@media (min-width: 768px)': {
    height: 80,
  },
});
