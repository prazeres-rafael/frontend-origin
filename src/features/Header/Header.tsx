import React from 'react';
import { Logo as LogoBase } from 'components';
import { HeaderOrigin } from './styled';

export default function Header(): React.ReactElement {
  return (
    <HeaderOrigin>
      <LogoBase />
    </HeaderOrigin>
  );
}
