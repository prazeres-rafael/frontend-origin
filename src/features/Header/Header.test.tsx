import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import Header from './index';

describe('Header', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
