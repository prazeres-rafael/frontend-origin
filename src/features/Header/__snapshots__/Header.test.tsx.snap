// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
.c0 {
  background-color: #FFF;
  height: 56px;
  left: 0;
  position: relative;
  top: 0;
  width: 100%;
}

@media (min-width:768px) {

}

@media (min-width:768px) {
  .c0 {
    height: 80px;
  }
}

<div>
  <header
    class="c0"
  >
    <logo-origin.svg
      classname="styled__LogoOrigin-sc-y50cdy-0 kbYqGp"
    />
  </header>
</div>
`;
